### ISSUE TITLE
the model cannot be trained

### ISSUE THREAD

**Issue by hht1996ok** [kaist-avelab/PointStack#2]
Thanks to the author's excellent work, I encountered the problem of abnormally low model accuracy when training scanobjectnn. It can be seen that the loss of the model is constantly decreasing, but Val Accuracy keeps abnormally

Val Loss:  22.731 Val Accuracy:  7.148 Val Avg Accuracy:  6.54
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:38<00:00,  1.04it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:20<00:00, 35.76it/s]
==================== Epoch 24 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2881/2882 [01:20<00:00, 43.31it/s]
Val Loss:  18.502 Val Accuracy:  7.425 Val Avg Accuracy:  5.038
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:53<00:00,  1.00it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:15<00:00, 38.00it/s]
==================== Epoch 25 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2878/2882 [01:15<00:00, 41.23it/s]
Val Loss:  14.118 Val Accuracy:  8.536 Val Avg Accuracy:  5.12
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:45<00:00,  1.02it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:10<00:00, 41.06it/s]
==================== Epoch 26 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2879/2882 [01:10<00:00, 41.51it/s]
Val Loss:  12.804 Val Accuracy:  9.715 Val Avg Accuracy:  8.381
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:59<00:00,  1.01s/it]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:09<00:00, 41.75it/s]
==================== Epoch 27 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2881/2882 [01:08<00:00, 43.67it/s]
Val Loss:  17.518 Val Accuracy:  8.119 Val Avg Accuracy:  6.131
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:46<00:00,  1.02it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:16<00:00, 37.91it/s]
==================== Epoch 28 ====================██████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:16<00:00, 33.55it/s]
Val Loss:  16.455 Val Accuracy:  6.315 Val Avg Accuracy:  5.099
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:51<00:00,  1.01it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:08<00:00, 42.10it/s]
==================== Epoch 29 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2879/2882 [01:08<00:00, 41.35it/s]
Val Loss:  14.071 Val Accuracy:  11.832 Val Avg Accuracy:  6.374
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:46<00:00,  1.02it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:21<00:00, 35.57it/s]
==================== Epoch 30 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2880/2882 [01:20<00:00, 41.89it/s]
Val Loss:  11.898 Val Accuracy:  7.078 Val Avg Accuracy:  5.705
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:46<00:00,  1.02it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:07<00:00, 42.38it/s]
==================== Epoch 31 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2879/2882 [01:07<00:00, 37.85it/s]
Val Loss:  9.717 Val Accuracy:  6.593 Val Avg Accuracy:  4.502
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:54<00:00,  1.00it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:13<00:00, 39.05it/s]
==================== Epoch 32 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2878/2882 [01:13<00:00, 40.75it/s]
Val Loss:  10.663 Val Accuracy:  10.548 Val Avg Accuracy:  6.144
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:47<00:00,  1.02it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:07<00:00, 42.64it/s]
==================== Epoch 33 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2880/2882 [01:07<00:00, 36.43it/s]
Val Loss:  9.87 Val Accuracy:  8.258 Val Avg Accuracy:  5.976
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:58<00:00,  1.01s/it]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:10<00:00, 40.69it/s]
==================== Epoch 34 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2881/2882 [01:10<00:00, 27.22it/s]
Val Loss:  11.58 Val Accuracy:  5.795 Val Avg Accuracy:  4.91
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:42<00:00,  1.03it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:16<00:00, 37.84it/s]
==================== Epoch 35 ====================█████████████████████████████████████████████████████████████████████████████████████████▉| 2881/2882 [01:16<00:00, 34.47it/s]
Val Loss:  9.412 Val Accuracy:  6.211 Val Avg Accuracy:  4.849
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:52<00:00,  1.00it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:20<00:00, 35.98it/s]
==================== Epoch 36 ====================█████████████████████████████████████████████████████████████████████████████████████████▊| 2879/2882 [01:20<00:00, 33.89it/s]
Val Loss:  10.826 Val Accuracy:  7.113 Val Avg Accuracy:  6.271
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 475/475 [07:39<00:00,  1.03it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2882/2882 [01:19<00:00, 36.44it/s]

My training parameters
    NAME: SGD
    MAX_EPOCH: 200
    BATCH_SIZE: 24
    GRAD_ACCUMULATION: 1
    GRAD_CLIP: 100
    LR: 0.005
    MIN_LR: 0.00005
    MOMENTUM: 0.9
    NESTEROV: True
    WEIGHT_DECAY: 0.00025
    BETAS: [0.9, 0.999] # For Adam
    SCHEDULER: cosine_annealing
    WARM_RESTART_EVERY: 200

**Comment by ktirta** [comment]
Hi @hht1996ok, thanks for your interest in our work.

Unfortunately, I couldn't reproduce this issue on our machine.
The val accuracy (left) and val loss (right) when I trained for 3 epochs with your training parameters look like the following,
<p float="left">
  <img src="https://user-images.githubusercontent.com/52604324/171087213-b008d238-a4cd-428f-98f3-fb583010dad6.svg" width="256" />
  <img src="https://user-images.githubusercontent.com/52604324/171087220-1e7113b2-bbe9-4418-802d-0f60f5f2d196.svg" width="256" /> 
</p>
Is there any other modifications to the codes / config parameters?

**Comment by hht1996ok** [comment]
@ktirta Thank you for your reply. I will try it in a different environment. @ktirta
